src/refac/__init__.py

```diff
@@ -1,7 +1,7 @@
 __description__ = "Tool to move around Python source code."
 __license__ = "MIT License"
 __uri__ = "https://github.com/benchling/refac"
-__version__ = "0.0.2"
+__version__ = "0.0.3"
```

Comment:
huh, is sticking private constants into `__init__.py` like this a python standard? should we add that to our package template? I like the idea of being able to identify package versions from code